packit/api.py

```diff
@@ -1700,6 +1700,14 @@ def _handle_sources(
         # therefore it needs the archive itself beforehand.
         upstream_archives = self.dg.download_upstream_archives()
 
+        # Filter out git-tracked upstream archives
+        untracked_upstream_archives = [
+            archive
+            for archive in upstream_archives
+            if str(archive.relative_to(self.dg.absolute_source_dir))
+            not in [p for p, _ in self.dg.local_project.git_repo.index.entries]
```

Comment:
could we extract getting of the git tracked files to a method, so that this is not duplicated (for local sources)?